views/index.ejs

```diff
@@ -71,6 +71,7 @@
     <div class="checkbox-wrapper"><label><input type="checkbox" checked id="teacher-toggle"><span>Show Teachers</span></label></div>
     <div class="checkbox-wrapper"><label><input type="checkbox" checked id="room-toggle"><span>Show Room</span></label></div>
     <div class="checkbox-wrapper"><label><input type="checkbox" checked id="free-toggle"><span>Show Free Periods</span></label></div>
+    <div class="checkbox-wrapper"><labeL><input type="checkbox" id="color-toggle"><span>Show Color Background</span></label></div>
```

Comment:
Are we not showing the colors by default?

Reply:
We can do that